### DISCUSSION TITLE
libssh2_sftp_open is too slow.

### DISCUSSION THREAD

**Discussion by SeongminJaden** [libssh2/libssh2#1120] [Q&A]
libssh2_sftp_open is too slow. The photo stored on the ftp server is 40 megabytes, and the download speed takes more than 1 minute and 40 minutes. Is there any way to speed up the transfer?

**Comment by SeongminJaden** [comment]
sftp_handle = libssh2_sftp_open(sftp, remoteFilePath, LIBSSH2_FXF_READ, 0);

**Comment by SeongminJaden** [comment]
this is my code

**Comment by willco007** [comment]
Opening a file isn't downloading it, it is only opening the remote file handle and should be nearly instant.  If it's not, I would suggest looking at the sftp server logs to see if you can tell why it's replying so slowly.
